### PR TITLE
Add new API functions to get associated devices

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Related to https://github.com/omichel/webots/pull/6361.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

File: reference/positionsensor.md [hunk 1]
```diff
@@ -63,3 +63,22 @@ return `WB_ANGULAR` if the sensor is associated with a
...
 `WB_LINEAR` if it is associated with a [SliderJoint](sliderjoint.md) or a
 [Track](track.md) node.
+
+---
+
+**Name**
+
+**wb\_position\_sensor\_get\_brake**, **wb\_position\_sensor_get\_motor** - *get associated devices*
+
+{[C++](cpp-api.md#cpp_motor)}, {[Java](java-api.md#java_motor)}, {[Python](python-api.md#python_motor)}, {[Matlab](matlab-api.md#matlab_motor)}, {[ROS](ros-api.md)}
```
Comment (omichel) [inline reference/positionsensor.md]: Shouldn't it be: 
```
{[C++](cpp-api.md#cpp_position_sensor)}, {[Java](java-api.md#java_position_sensor)}, {[Python](python-api.md#python_position_sensor)}, {[Matlab](matlab-api.md#matlab_position_sensor)}, {[ROS](ros-api.md)}
```
instead?

Comment (stefaniapedrazzi) [reply reference/positionsensor.md]: Yes, I will fix the brake too.

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
Yes, it was.

**Comment by stefaniapedrazzi** [comment]
I updated the documentation and removed the "const" attribute from the C++ functions to match the code.
:+1:

File: reference/java-api.md [hunk 2]
```diff
@@ -600,6 +604,8 @@ The following tables describe the Java classes and their methods.
 | &nbsp;&nbsp; public int [getSamplingPeriod](positionsensor.md#wb_position_sensor_get_value)();          |
 | &nbsp;&nbsp; public double [getValue](positionsensor.md#wb_position_sensor_get_value)();                |
 | &nbsp;&nbsp; public int [getType](positionsensor.md#wb_position_sensor_get_value)();                    |
+| &nbsp;&nbsp; public [Brake](#java_brake) [getBrake](positionsensor.md#wb_position_sensor_get_brake)();                 |
+| &nbsp;&nbsp; public [Motor](#java_motor) [getMotor](positionsensor.md#wb_position_sensor_get_motor)();                 |
```
Comment (omichel) [inline reference/java-api.md]: Minor detail: indentation is not very nice here.

File: reference/motor.md [hunk 3]
```diff
@@ -535,3 +535,22 @@ WB\_LINEAR, and otherwise it returns WB\_ANGULAR.
...
+```c
+#include <webots/brake.h>
+
+WbDeviceTag wb_motor_get_brake(WbDeviceTag tag);
+WbDeviceTag wb_motor_get_position_sensor(WbDeviceTag tag);
+```
+
+**Description**
+
+The `wb_motor_get_motor()` and `wb_motor_get_position_sensor` functions return the [Brake](brake.md) and [PositionSensor](positionsensor.md) instances defined in the same [Joint](joint.md) or [Track](track.md) `device` field. If none is defined they return 0.
```
Comment (omichel) [inline reference/motor.md]: `wb_motor_get_motor()` seems wrong.

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**